### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by Karan-Jain**

### CODE REVIEW

File: .github/pull_request_template.md
Comment (borisliao) [inline .github/pull_request_template.md]:
```suggestion
```

We probably don't need these sections now.

**Review by borisliao — CHANGES_REQUESTED** [review]

**Review by borisliao — DISMISSED** [review]

File: .github/pull_request_template.md
Comment (Exsects) [inline .github/pull_request_template.md:34]: Maybe we can add a field to schedule a meeting with the team to see if the PR is impacting other areas of the code.

File: .github/pull_request_template.md
Comment (Exsects) [inline .github/pull_request_template.md]:
```suggestion
```

I don't think we need this since setting the PR as a draft already replaces the last two options. And the PR is already in ready state if it's waiting for approval.

File: .github/pull_request_template.md
Comment (Exsects) [inline .github/pull_request_template.md:32]:
```suggestion
- [ ] Updated Tests
```
Doesn't the CI pipeline already check the quality check and tests pass?

File: .github/pull_request_template.md
Comment (Exsects) [inline .github/pull_request_template.md]: I think we can move this to a todo to make sure people are connecting the PR with the issues they are working on.

File: .github/pull_request_template.md
Comment (Exsects) [inline .github/pull_request_template.md]: I think we can add this to the Todo and remind people to use labels when creating their PRs.

**Review by Exsects — DISMISSED** [review]

File: .github/pull_request_template.md
Comment (Karan-Jain) [reply .github/pull_request_template.md]: I think its better to keep the todo separate and the changes separate. 
TODO's are basically code  sanitizing steps - I will just change the naming

File: .github/pull_request_template.md
Comment (Karan-Jain) [reply .github/pull_request_template.md:32]: The passed quality checks is basically a reminder for us to run quality checking tools locally or do some initial quality checking locally

File: .github/pull_request_template.md
Comment (Karan-Jain) [reply .github/pull_request_template.md]: Makes Sense

File: .github/pull_request_template.md
Comment (Karan-Jain) [reply .github/pull_request_template.md:34]: Is there a way to do that using links?

Comment (Exsects) [reply .github/pull_request_template.md:34]: Not sure what you mean by links.

File: .github/pull_request_template.md
Comment (Exsects) [reply .github/pull_request_template.md]: I'll commit the changes while I'm here.

File: .github/pull_request_template.md
Comment (Karan-Jain) [reply .github/pull_request_template.md:34]: Lets keep it status quo, and we can research on it later. I don't think there is a field for scheduling, 
unless, you have any suggestions....

**Review by borisliao — DISMISSED** [review]

**Review by Exsects — APPROVED** [review]

**Review by borisliao — APPROVED** [review]

**Pull request merged**
